billing/src/Filament/App/Resources/Orders/Pages/CheckoutPage.php

```
@@ -0,0 +1,108 @@
+            return;
+        }
+
+        $this->selectedMethod = $method;
+        $this->order->update(['payment_method' => $method]);
+    }
+
+    public function proceedToStripe(): void
+    {
+        try {
+            /** @var StripeClient $stripeClient */
+            $stripeClient = app(StripeClient::class);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$stripeClient'. (undefined)

(UnusedLocalVariable)